Lab1/app/routes/route_test.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint
+
+from app.models.User import User
+
+test_blueprint = Blueprint('route_test', __name__, url_prefix='/test')
+
+
+@test_blueprint.route('/testlogin')
```

Comment:
Sample router for request "/test/testlogin", will simply reply a json containing all the information of the current user. 

This route simply shows the use case of return json files and could be deleted since we are dealing with one user only.